package.json

```diff
@@ -30,19 +29,27 @@
   "eslintIgnore": [
     "build/*"
   ],
-  "pico-check": {
-    "require": "./tests/test.init.js"
+  "jest": {
+    "modulePaths": [
+      "node_modules",
+      "<rootDir>"
+    ],
+    "preset": "@shelf/jest-mongodb"
   },
   "babel": {
     "presets": [
-      "env",
-      "react"
+      "@babel/preset-env",
+      "@babel/preset-react"
```

Comment:
I'm assuming this is functionally identical, right? Just using the more "correct" naming scheme here?

Reply:
I assume the same, but I'm not so familiar with all that stuff to say for sure